compose.yml

```diff
@@ -0,0 +1,19 @@
+# This file is used to run a local instance of
+# Prefect for testing our Terraform Provider.
+#
+# See ./_about/CONTRIBUTING.md.
+services:
+  prefect:
+    image: prefecthq/prefect:2-latest
```

Comment:
i dont think it's out yet, but we may end up updating this to `prefect:3-latest`

Reply:
True, good callout.